src/Analyzers/CSharp/Tests/ConvertSwitchStatementToExpression/ConvertSwitchStatementToExpressionTests.cs

```diff
@@ -1941,145 +2097,153 @@ class Program
+                """);
         }
 
         [Fact]
         public async Task TopLevelStatement()
         {
-            var source = @"
-int i = 0;
-[|switch|] (i)
-{
-    case 1:
-        return 4;
-    default:
-        return 7;
-}";
-
-            var fixedSource = @"
-int i = 0;
-return i switch
-{
-    1 => 4,
-    _ => 7,
-};
-";
+            var source = """
+                int i = 0;
+                [|switch|] (i)
+                {
+                    case 1:
+                        return 4;
+                    default:
+                        return 7;
+                }
+                """;
+
+            var fixedSource = """
+                int i = 0;
+                return i switch
+                {
+                    1 => 4,
+                    _ => 7,
+                };
+
+                """;
 
             var test = new VerifyCS.Test
             {
-                TestCode = source,
+                TestState =
+                {
+                    OutputKind = OutputKind.ConsoleApplication,
+                    Sources = { source },
+                },
                 FixedCode = fixedSource,
                 LanguageVersion = LanguageVersion.CSharp9,
             };
 
-            test.ExpectedDiagnostics.Add(
-                // /0/Test0.cs(2,1): error CS8805: Program using top-level statements must be an executable.
-                DiagnosticResult.CompilerError("CS8805").WithSpan(2, 1, 2, 11));
-
             await test.RunAsync();
         }
 
         [Fact, WorkItem(44449, "https://github.com/dotnet/roslyn/issues/44449")]
         public async Task TopLevelStatement_FollowedWithThrow()
         {
             // We should be rewriting the declaration for 'j' to get 'var j = i switch ...'
-            var source = @"
-int i = 0;
-int j;
-[|switch|] (i)
-{
-    case 1:
-        j = 4;
-        break;
-    case 2:
-        j = 5;
-        break;
-}
-throw null;
-";
+            var source = """
+                int i = 0;
+                int j;
+                [|switch|] (i)
+                {
+                    case 1:
+                        j = 4;
+                        break;
+                    case 2:
+                        j = 5;
+                        break;
+                }
+                throw null;
+                """;
 
-            var fixedSource = @"
-int i = 0;
-int j;
-j = i switch
-{
-    1 => 4,
-    2 => 5,
-    _ => throw null,
-};
-";
+            var fixedSource = """
+                int i = 0;
+                int j;
+                j = i switch
+                {
+                    1 => 4,
+                    2 => 5,
+                    _ => throw null,
+                };
+
+                """;
 
             var test = new VerifyCS.Test
             {
-                TestCode = source,
+                TestState = {
+                    Sources = { source },
+                    OutputKind = OutputKind.ConsoleApplication,
+                },
```

Comment:
❓ Why is a mechanical factoring PR containing code changes? This should not have changed.

Reply:
so changing the test here just changed diagnostic spans.  That's pointless pain (since this test is not seeking to verify that you get a "Program using top-level statements must be an executable." error).  So this was changed to remove the error, and just pass teh flag to teh compiler that ensures we test the thing we care about, without extraneous (*position-sensitive*) data in it.